### ISSUE TITLE
Lyso-sphingomyelin high in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11240]
**Preferred term label:**
Lyso-sphingomyelin high in blood

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Niemann-Pick disease type A
Niemann-Pick disease type B

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001380
- label: Elevated circulating lysosphingomyelin concentration
- parents: [Abnormal circulating lipid concentration (HP:0003119)](https://hpo.jax.org/app/browse/term/HP:0003119)
- pmid: PMID:36348386
- def: The concentration of lysosphingomyelin in the blood circulation is above the upper limit of normal. Acid sphingomyelinase deficiency (ASMD) or Niemann-Pick type A/B is a lysosomal disorder caused by the deficiency of acid sphingomyelinase (ASM) due to pathogenic variants in the SMPD1 gene. Sphingomyelin (SM) is the substrate for ASM that cleaves the phosphorylcholine linkage of SM producing ceramide. SM is a major compound of most cell membranes and coupled with cholesterol constitutes most of the membrane rafts. In the deficiency of ASM, there is primary storage of SM, and the concentration of lysosphingomyelin in dried blood spots and in the blood circulation is typically elevated.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11240
- synonyms:
